Atos20/fitlit-project
# User class test implementation

## PR Template:
### What’s this PR do?  
-  This Pr creates a test and implementation file for the user class
- This user class is able to instantiate a user with an `id, name, address, email, strideLength, dailyStepGoal, friends`
- it can also return the user's first name

### Where should the reviewer start?  
 Open user-test.js and test.js with a text editor 

### How should this be manually tested? 
run  `npm test  test/user-test.js` on the terminal
open user-test.js and user.js with a text editor to see at the code implemented 
 
### Any background context you want to provide?  
n/a
### What are the relevant tickets?  
#36, #11, #4 and #6

### Screenshots (if appropriate) 

![Screen Shot 2020-08-24 at 6 19 42 PM](https://user-images.githubusercontent.com/56229864/91108852-62e4b680-e636-11ea-8602-1b9db8656dad.png)
 
#### Questions:
